vmdb/app/helpers/application_helper.rb

```diff
@@ -63,7 +63,8 @@ def hidden_tag_if(tag, condition, options = {}, &block)
 
   # Check role based authorization for a UI task
   def role_allows(options={})
-    role_allows_intern(options) rescue false
+    store = Vmdb::PermissionStores.instance
+    role_allows_intern(options) && store.can?(options[:feature]) rescue false
```

Comment:
Ya.  I'm actually going to pull this change out.  I had a discussion with @dclarizio and we want to be more fine grain.  In other words, change the places that need to check for permissions (the tab locations, etc) via a different method.  I'm working on updating the PR now.